### ISSUE TITLE
iSC/Cold chain: fixed range for date filters excludes breaches that might be ongoing

### ISSUE THREAD

**Issue by marthakeezy** [msupply-foundation/open-msupply#4251]
## What went wrong? 😲

**Cold chain -> Monitoring**

When accessing Monitoring, by default the date filter will only show the last 24 hours. While date filter is editable, the first glance the user gets would be the last 24 hours. The filter value remains the same across all three tabs - Monitoring, Breaches and Log.

**Cold chain -> Breaches**

Because of the filter described above, the breaches actually do not show up unless they happened in the last 24 hours. So the beginning time of the breach has to fall within the date range that is set by default for the user to see it straight away without having to change any details. 

I wonder if it creates a false picture of the site having no breaches. Because the first screen the user sees is 'No Temperature Breaches Found' and the users might not even realise they have date filters on or might not pay attention. I definitely got caught by this the first time I used Cold chain after the filters were fixed, and it took me solid 10 minutes to figure out breaches don't show up because of the filter. 

## Expected behaviour 🤔

I think it would be **really** good to either: 

- Always show the ongoing breaches, regardless of the date filter. Maybe Unacknowledged breaches should always be shown too. 
- Or remove the default date setting specifically for 'Breaches' tab to allow all breaches to be shown. 

@adamdewey might have more thoughts on what would be best for the user :) I personally think default date filter would be good to minimise the information shown on the screen straight away, but we need to have exceptions for the ongoing breaches or those, that haven't been acknowledged. 

## How to Reproduce 🔨

Steps to reproduce the behaviour:

1. Produce a breach that is older than 24 hours
2. Go to Cold chain -> Monitoring
3. Note the default date/time in the date filter
4. Navigate to Breaches -> see the breach that is older than your time range not appear
5. Confusing!

## Your environment 🌱

- Open mSupply Version: V2.1.0-RC2
- Legacy mSupply Central Server Version: V7.15.05

- Platform: Desktop App on Windows

- Database type: SQLite3

**Comment by andreievg** [comment]
Triage, waiting for @adamdewey to decide on severity

**Comment by adamdewey** [comment]
Yes, agree - the tab should show breaches that have any part of them that fall within the current date range. 

Should apply to both ongoing and ended.

**Comment by adamdewey** [comment]
Severity Normal is good

**Comment by andreievg** [comment]
Functionally makes sense to include ongoing breach. Maybe a little bit harder on technical end, because of end date time not populated before breach is finished.

**Comment by mlenignol** [comment]
@msupply-foundation/triage Valid to include ongoing breaches - if this haven't been addressed yet I'd keep it
